browser/javascript/calculatorHelpers/showNumberOnScreen.js

```diff
@@ -0,0 +1,10 @@
+function showNumberOnScreen(value, calculator){
+  var screen = document.querySelector(`div#${calculator.name} div.calculator-screen-number`)
+  var history = document.querySelector(`div#${calculator.name} div.calculator-screen-history`)
+
+  screen.innerHTML = calculator[value]
```

Comment:
The way that `showNumberOnScreen` accepts a value like "previous" and then passes it into calculator here is brilliant.